providers/aws/sg.go

```diff
@@ -15,41 +15,221 @@
 package aws
 
 import (
+	"bytes"
 	"context"
-	"strings"
-
+	"fmt"
 	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
-
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/hashicorp/terraform/flatmap"
+	"gonum.org/v1/gonum/graph"
+	simplegraph "gonum.org/v1/gonum/graph/simple"
+	"gonum.org/v1/gonum/graph/topo"
+	"sort"
+	"strings"
 )
 
 var SgAllowEmptyValues = []string{"tags."}
 
+type void struct{}
+
+var member void
+
 type SecurityGenerator struct {
 	AWSService
 }
 
+type ByGroupPair []ec2.UserIdGroupPair
+
+func (b ByGroupPair) Len() int      { return len(b) }
```

Comment:
Why not use?
```
sort.Slice(planets, func(i, j int) bool {
  return planets[i].Axis < planets[j].Axis
})
```

Reply:
I just referred to how it was done in the provider https://github.com/terraform-providers/terraform-provider-aws/blob/f61b73d3976a2f9d1afe77d43f604b2f2f25a063/aws/resource_aws_security_group_rule.go#L448